dvc/remote/gdrive.py

```diff
@@ -246,7 +246,7 @@ def gdrive_upload_file(
         self,
         parent_id,
         title,
-        no_progress_bar=True,
+        no_progress_bar=False,
```

Comment:
I suppose these were here (and in gs down below) because they didn't support pbars earlier and we forgot to adjust them here. Nice catch!